### PR TITLE
Add method for groups.setAnnouncement

### PR DESCRIPTION
**Pull request by BenjaminBoehm**
https://developer.rocket.chat/reference/api/rest-api/endpoints/team-collaboration-endpoints/groups-endpoints/setannouncement

### CODE REVIEW

File: rocketchat_API/APISections/groups.py [hunk 1]
```diff
@@ -106,6 +106,15 @@ def groups_rename(self, room_id, name, **kwargs):
             "groups.rename", roomId=room_id, name=name, kwargs=kwargs
         )
 
+    def groups_set_announcement(self, room_id, announce, **kwargs):
```
Comment (jadolg) [inline rocketchat_API/APISections/groups.py:109]: I'd like to keep it as close to the API naming as possible as long as we don't break Python's style
```suggestion
    def groups_set_announcement(self, room_id, announcement, **kwargs):
```

File: rocketchat_API/APISections/groups.py [hunk 2]
```diff
@@ -106,6 +106,15 @@ def groups_rename(self, room_id, name, **kwargs):
...
+        """Sets the announcement for the private group."""
+        return self.call_api_post(
+            "groups.setAnnouncement",
+            roomId=room_id,
+            announcement=announce,
```
Comment (jadolg) [inline rocketchat_API/APISections/groups.py:114]:
```suggestion
            announcement=announcement,
```

**Review by jadolg — CHANGES_REQUESTED** [review]

**Comment by BenjaminBoehm** [comment]
Then we should consider changing the `channels_set_announcement` argument as well to keeping API naming:
https://github.com/jadolg/rocketchat_API/blob/5ef7fba34f140673726d3c8bfb4ac87bc563b93c/rocketchat_API/APISections/channels.py#L176

Reference: https://developer.rocket.chat/reference/api/rest-api/endpoints/team-collaboration-endpoints/channels-endpoints/setannouncement

**Comment by jadolg** [comment]
> Then we should consider changing the `channels_set_announcement` argument as well to keep API naming:
> 
> https://github.com/jadolg/rocketchat_API/blob/5ef7fba34f140673726d3c8bfb4ac87bc563b93c/rocketchat_API/APISections/channels.py#L176
> 
> Reference: https://developer.rocket.chat/reference/api/rest-api/endpoints/team-collaboration-endpoints/channels-endpoints/setannouncement

nice catch :)

**Pull request merged**